api/app/signals/apps/signals/models/signal.py

```diff
@@ -48,6 +48,10 @@ class Signal(CreatedUpdatedModel):
                                     related_name='signal',
                                     null=True,
                                     on_delete=models.SET_NULL)
+    directing_departments_assignment = models.OneToOneField('signals.DirectingDepartments',
+                                                            related_name='signal',
```

Comment:
I realize we do this in more places, so take this as something we may want to address outside of this PR, but:

For the most recent `DirectingDepartments` instance we can use both `signal` and `_signal` to get the Signal instance associated with it. I don't think this is the behavior we want. (We only ever use `_signal` in this case as far as I know.)

Reply:
Did this to keep it consistent with the rest of the code in the `Signal` model. So if we want to address this it is out of scope for this PR